dwa_local_planner/src/dwa_planner_ros.cpp

```diff
@@ -518,15 +520,39 @@ namespace dwa_local_planner {
   void DWAPlannerROS::actuator_position_callback(const lexxauto_msgs::ActuatorStatus::ConstPtr& msg)
   {
     this->is_actuator_connect_ = msg->connect;
+
+    // The decision should be delegated to the node responsible for Cargo Status at some point. => AMRCS-174
+    this->is_cargo_fixed_ = false;
+    if (this->cargo_mode_ == "wani")
+    {
+      if (actuator_status::ACT_MID2 <= msg->position[0])
+      {
+        this->is_cargo_fixed_ = true;
+      }
+    }
+    else if (this->cargo_mode_ == "towing")
+    {
+      if (!msg->connect)
+      {
+        this->is_cargo_fixed_ = true;
+      }
+    }
+    else if (this->cargo_mode_ == "loading")
+    {
+      this->is_cargo_fixed_ = true;
+    }
   }
 
   void DWAPlannerROS::cargo_angle_callback(const std_msgs::Float64::ConstPtr& msg)
   {
     this->dp_->setCargoAngle(msg->data);
     this->goalLatchedStopRotateController_.setCargoAngle(msg->data);
     this->startLatchedStopRotateController_.setCargoAngle(msg->data);
-    this->is_cargo_enabled_ = true;
-    this->cargo_angle_recv_time_ = ros::Time::now();
+    if (!this->is_cargo_fixed_)
+    {
+      this->is_cargo_enabled_ = true;
+      this->cargo_angle_recv_time_ = ros::Time::now();
+    }
```

Comment:
以下は不要ですか？
```
else
{
  this->is_cargo_enabled_ = false;
}
```

Reply:
It looks like it would be better to have it to increase response after lowering the linear actuator.
If there is no need to be in such a hurry, it seems to revert to false by not being received for a certain period of time.
at `DWAPlannerROS::check_cargo_angle()`

リニアアクチュエータを下げたあとのレスポンスを上げるためにはあった方がよいように見えます。
そこまで急がなくて良いのであれば、一定時間未受信によってfalseに戻るようですね。
at `DWAPlannerROS::check_cargo_angle()`

Reply:
else を追加しました。

Reply:
ありがとうございます。リリースして ckd で使ってもらうので、万が一のために最後にテストしといた方が良いと思います。いそがしければ僕やります（最低限 loading だけでも

**loading**
- [ ] act low 回転が path 計算に影響する
- [ ] act high 回転が path 計算に影響する

wani
- [ ] center act mid2 で回転が path 計算に影響する
- [ ] center act low で回転が path 計算に影響しない

Reply:
それぞれ1回づつ確認してマージによるデグレが起きていないことを確認しました。